js/main.js

```diff
@@ -59,36 +59,39 @@ function renderEntry(entry) {
   return list;
 }
 
-var $ul = document.querySelector('ul');
+function toggle(event) {
+  if (event.target.value === 'NEW') {
+    event.preventDefault();
+    $viewForm.setAttribute('class', 'view-form');
+    $viewEntries.setAttribute('class', 'view-entries hidden');
+    data.view = 'form';
+  }
+  if (event.target.textContent === 'Entries') {
+    event.preventDefault();
+    $viewForm.setAttribute('class', 'view-form hidden');
+    $viewEntries.setAttribute('class', 'view-entries');
+    data.view = 'entries';
+  }
+}
```

Comment:
This is not maintainable, you do not want your JavaScript to be reliant on a specific HTML structure or even worse, a specific text content for an element. Remember that JavaScript should be our one source of truth, it shouldn't rely on data from HTML to decide what it needs to do. I stand behind creating a single function named something like `switchView` that receives an argument of the name of the view to switch to, and the function should show that view and hide all other views. That way, you won't need to make a new function for each new view created. or a new conditional block in this function